.github/workflows/ruby-tests.yml

```diff
@@ -0,0 +1,123 @@
+name: Ruby Tests
```

Comment:
Please move the tests from here to spec files. We have existing verify pipelines which will tc of running these specs

Reply:
I see the tests are added in the spec. We don't need this.